src/gurt/tests/SConscript

```diff
@@ -1,23 +1,7 @@
-# (C) Copyright 2020 Intel Corporation.
+# (C) Copyright 2021 Intel Corporation.
```

Comment:
2020-2021...Actually, parts of this code are much older than that

Reply:
Hmm, perhaps moved to this location last year, or newly created?
b6925434 (David M. Moreno 2020-08-03 17:55:24 -0500  4) #
b6925434 (David M. Moreno 2020-08-03 17:55:24 -0500  5) """Unit tests"""